src/components/BulkMetadataForm/BulkMetadataFormStyles.ts

```diff
@@ -0,0 +1,89 @@
+import styled from 'styled-components';
+import { themeSpacing } from '@amsterdam/asc-ui';
+
+const BulkMetadataFormStyles = styled.div`
+	input[type='text'] {
+		margin: ${themeSpacing(3, 0, 6)};
+	}
+
+	.MuiGrid-item {
+		text-align: center;
+	}
+
+	.MuiGrid-spacing-xs-2 div:nth-of-type(2) {
```

Comment:
No need to split this into 2 levels here.

Reply:
Fixed